### ISSUE TITLE
Enhance the contrast of the warning icon

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#169]
### Problem
[atoms/atoms.input](https://a-ui.github.io/core_branding_scss/#atoms-input)

If the warning icon is not accompanied by a text label, it is not decorative. Informative user interface elements need to have a minimum contrast ratio of 3:1. The current yellow on white fails with a ratio of 1,9:1.

![screenshot warning icon](https://user-images.githubusercontent.com/870049/98813672-dec7f100-2424-11eb-84e2-95169229b5b6.jpg)

### Solution 

Make the exclamation mark darker (black) to create a higer contrast with the surrounding yellow.

### Links and articles

[Understanding Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast)

**Comment by TriangleJuice** [comment]
Fixed in #183
